MetaMask/metamask-mobile
# feat: Network verification changed

## **Description**
This PR aims to update network verification UI to help the user with more warnings when adding a custom network via dapp.


## **Related issues**

Fixes: #

## **Manual testing steps**

1. Go to this chainlist.org on in app browser
2. Add a custom network
3. Should see the new layout

## **Screenshots/Recordings**
 * This recording have all the warnings for testing purposes
https://recordit.co/QTde2ZiVrv

redirect to add custom network article
https://recordit.co/U7eo17r4qD
https://recordit.co/zywbsTqOec
### **Before**

### **After**
https://recordit.co/QTde2ZiVrv


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
